spec/features/post/post_index_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
```

Comment:
- [ ] You should have a test case that asserts you can see the user's profile picture.

Reply:
Done ✔